src/views/TeamPage/components/Subsystems/Subsys.css

```diff
@@ -0,0 +1,103 @@
+.subsystem-section {
+  padding-top: 40px;
+}
+
+.cardbody {
+  padding: 10px;
+}
+
+.member-img {
+  width: 100%;
+  max-width: 350px;
+}
+
+.card-background {
+  width: 100%;
+  padding: 15px;
+  margin-bottom: 20px;
+  box-shadow: 0px 2px 15px rgba(53, 35, 35, 0.2);
+}
+
+.subsystem-text {
+  font-size: 18px;
+  font-family: var(--font-primary);
+  font-weight: 600;
+  /* color: var(--color-secondary); */
```

Comment:
The default text is too faint. Change it color-primary.

Reply:
done